src/Guides/RestructuredText/HTML/Directives/Uml.php

```diff
@@ -56,4 +73,25 @@ public function wantCode(): bool
     {
         return true;
     }
+
+    private function loadExternalUmlFile(Environment $environment, string $path): ?string
```

Comment:
if I get this method correctly, It will load the contents of the uml as it was inline? So the rendering will pick this up?

Why was the absolute path needed for this? I would expect that flysystem can still load the file? This won't work when a file is outside the root of the documentation dir. I think this would be a valid constraint.

Reply:
> if I get this method correctly, It will load the contents of the uml as it was inline? So the rendering will pick this up?

This is correct; our PlantUML renderer adds a bit of styling and it will thus follow the process like any other UML diagram.

> Why was the absolute path needed for this? I would expect that flysystem can still load the file? 

It is the absolute path on the Origin filesystem and not on the physical filesystem; the origin filesystem's root is at the project folder, but the filename in the environment is relative to the Guide's root. Thus a translation is needed from the Guide root to the Project root